src/librustdoc/clean/inline.rs

```diff
@@ -227,6 +227,18 @@ fn build_type(cx: &DocContext, tcx: &TyCtxt, did: DefId) -> clean::ItemEnum {
     }, false)
 }
 
+fn is_item_doc_reachable(cx: &DocContext, did: DefId) -> bool {
+    use ::visit_lib::LibEmbargoVisitor;
+    // FIXME: this is only partially correct for local items
+    //        (we're reusing the access levels from crateanalysis,
+    //         which does not considering `doc(hidden)`ness)
```

Comment:
After thinking more about this, this probably doesn't matter since an impl in an external crate can't reference a local type or trait (right??).